addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,219 @@
+          }
+          border-color: var(--failure);
+        }
+        &:checked {
+          opacity: 0;
+          ~ .rose-form-radio-label {
+            &::after {
+              position: absolute;
+              bottom: $size-xxs + $size-xxxs;
+              right: sizing.rems(m) * 10.9;
+              display: block;
+              width: $m;
+              height: $m;
+              transform: scale(2);
+              pointer-events: none;
+              content: ' ';
+              background: no-repeat 50%/50% 50%;
+              background-image: url($data-svg-prefix + map-get(
+                  $radio-svg,
+                  'error-checked'
+                ));
+              z-index: 2;
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.rose-form-radio-card-tile {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: ($m) * 15;
+  justify-content: center;
+
+  .radio-card-overlay {
+    position: absolute;
+    height: $l + $size-xxs;
+    background-color: var(--ui-gray-subtler-5);
+    width: 100%;
+    bottom: 0;
+    right: 0;
+  }
+  .rose-form-radio-label {
+    font-style: normal;
+    font-weight: bold;
+    font-size: typography.font-size(s);
+    color: var(--black);
+    width: ($m) * 15;
+    height: ($m) * 9;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    text-align: left;
+    word-break: break-all;
+    border: $xxxxs solid var(--ui-border);
+    box-shadow: 0 $size-xxs $size-xxs
+      rgba(var(--stark-components), var(--opacity-2));
+    border-radius: $size-xxxs;
+    padding: $m $size-xxs;
+    box-sizing: border-box;
+    background-color: var(--white);
+    .rose-form-helper-text {
+      font-weight: normal;
+    }
+  }
+  .rose-form-radio-field {
+    position: absolute;
+    z-index: 1;
+    bottom: $size-xs;
+    &:not(:disabled) {
+      &:hover {
+        ~ .rose-form-radio-label {
+          border: $xxxxs solid var(--ui-gray);
+        }
+      }
+      &:checked {
+        ~ .rose-form-radio-label {
+          .radio-card-overlay {
+            background-color: var(--action-subtler-1h);
+          }
+          border: $xxxxs solid #1563ff;
+        }
+      }
+    }
+    &:disabled {
+      ~ .rose-form-radio-label {
+        border: $xxxxs solid var(--ui-border-subtler-1);
+        background-color: var(--ui-gray-subtler-6);
+        color: var(--ui-gray-starker-1);
+      }
+    }
+  }
+
+  &.error {
+    .rose-form-radio-field {
+      &:not(:disabled):not(:focus) {
+        ~ .rose-form-radio-label {
+          .radio-card-overlay {
+            background-color: var(--failure-subtler-1);
+          }
+          border-color: var(--failure);
+        }
+        &:checked {
+          opacity: 0;
+          ~ .rose-form-radio-label {
+            &::after {
+              position: absolute;
+              bottom: $size-xxs;
+              right: sizing.rems(m) * 6.8;
```

Comment:
Avoid hardcoding a measurement number (use sizing).
Avoid using odd numbers for sizing.